docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,323 @@ task.download_outputs()
+    ln: ./lnInclude
+```
+
+This indicates an issue loading the dynamically compiled shared object file for
+the simulation. However, in the stdout file, the creation of a new library during
+runtime is confirmed:
+
+```
+Creating new library in "dynamicCode/_6a0e8d329f43dbbd4e13ef75c75be041b26176fc/platforms/linux64GccDPInt32Opt/lib/libcodeStream_6a0e8d329f43dbbd4e13ef75c75be041b26176fc.so"
+```
+
+This dynamic compilation feature allows for runtime adjustments and optimizations,
+contributing to the flexibility and efficiency of OpenFOAM simulations in
+high-performance computing environments.
+
+### Prerequisites
+
+Before running the simulation, ensure you have downloaded the input files. You
+can get them from the following
+[repository](https://develop.openfoam.com/committees/hpc/-/tree/develop/compressible/rhoPimpleFoam/LES/highLiftConfiguration).
+Once downloaded, move them into a directory named `highLiftConfiguration` inside
+your working folder.
+
+Here’s an example of how your directory structure should look after downloading
+the necessary files:
+
+```bash
+ls -lasgo highLiftConfiguration
+total 104
+ 0 drwxrwxr-x@ 14     448 Sep 23 11:44 .
+ 0 drwx------@ 19     608 Sep 23 11:49 ..
+ 0 drwxrwxr-x@ 12     384 Sep 20 09:43 0.orig
+ 8 -rwxr-xr-x@  1     626 Jun 13 10:09 Allclean
+16 -rwxr-xr-x@  1    6998 Jun 13 10:09 Allrun
+ 8 -rw-rw-r--@  1     991 Jun 13 10:09 COPYING
+48 -rw-rw-r--@  1   21547 Jun 13 10:09 README.md
+ 0 -rw-rw-r--@  1       0 Jun 13 10:09 case.foam
+ 0 drwxrwxr-x@  6     192 Sep 20 09:43 constant
+ 0 drwxrwxr-x@ 13     416 Jun 13 10:09 figures
+ 0 drwxrwxr-x@ 28     896 Sep 20 09:43 system
+24 -rw-rw-r--@  1   11399 Jun 13 10:09 thumbnail.png
+```
+
+### Modifications to the Simulation Parameters
+
+To make the simulation run faster, we made some adjustments to the original
+parameters. Specifically, we modified the time step (`dt`) to 0.00002, the
+initial time to 0.10, and the final time to 0.30. These changes help reduce the
+overall simulation time while maintaining reasonable accuracy for this high-lift
+configuration case.
+
+### Configuring and Running the Simulation
+
+We'll run the simulation on a virtual machine with 180 CPUs using the
+`c3d-standard-180` machine type. The simulation involves several preprocessing
+steps, including mesh generation (`blockMesh`, `snappyHexMesh`), and finally,
+the simulation run itself. 
+
+Below is the Python code to configure and run the OpenFOAM simulation:
+
+```python notest
+import inductiva
+import os
+
+machine_group = inductiva.resources.MachineGroup(machine_type="c3d-standard-180")
+machine_group.start()
+
+import inductiva
+
+# Set simulation input directory
+input_dir = "/path/to/highLiftConfiguration"
+
+# Read the simulation commands
+with open(os.path.join(input_dir,'commands.txt'), 'r') as file:
```

Comment:
Before presenting this script, show the commands file. Remember that you are writing for OpenFOAM experts, so 1st give them something that they are confortable with.